sql/core_clients_last_seen_raw_v1.sql

```diff
@@ -0,0 +1,86 @@
+  -- Equivalent to, but more efficient than, calling udf_bitmask_range(1, 28)
+CREATE TEMP FUNCTION
+  udf_bitmask_lowest_28() AS (0x0FFFFFFF);
+  --
+CREATE TEMP FUNCTION
+  udf_shift_one_day(x INT64) AS (IFNULL((x << 1) & udf_bitmask_lowest_28(),
+      0));
+  --
+CREATE TEMP FUNCTION
+  udf_combine_days(prev INT64,
+    curr INT64) AS (udf_shift_one_day(prev) + IFNULL(curr,
+      0));
+  --
+WITH
+  _current AS (
+  SELECT
+    -- In this raw table, we capture the history of activity over the past
+    -- 28 days for each usage criterion as a single 64-bit integer. The
+    -- rightmost bit represents whether the user was active in the current day.
+    CAST(TRUE AS INT64) AS days_seen_bits,
+    DATE_DIFF(submission_date, profile_date, DAY) AS days_since_created_profile,
+    CAST(NULL AS BOOLEAN) AS ping_seen_within_6_days_of_profile_creation,
```

Comment:
@jmccrosky - Are you aware of any investigations of new profile definition for nondesktop? I'm including the necessary bits here to do the same "within 7 days" calculation of valid new profiles as we do for desktop, but I don't know whether that's necessary. Perhaps we have better confidence in this value?

Reply:
I'm actually not sure.  Su is just proposing a new new-profile definition for Desktop, and we should do some work to figure out what's appropriate for mobile.  For now, I would extend the logic if it's not difficult.  This all may change significantly a some point, but we won't have any good idea any time in the short-term.

Reply:
I'll leave this as-is, then, and we can experiment.